packages/packages.csv

```diff
@@ -79,7 +79,7 @@ Microsoft.AspNetCore.Mvc.Formatters.Xml,ship,include
 Microsoft.AspNetCore.Mvc.Localization,ship,include
 Microsoft.AspNetCore.Mvc.Razor,ship,include
 Microsoft.AspNetCore.Mvc.Razor.Extensions,ship,include
-Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X,ship,include
+Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X,shipoob,exclude
```

Comment:
Are we actually shipping this as a package? I thought it was meant for VS only.

Reply:
Guess I misunderstood what we intend to do with this package. :shipit: 

Make sure you don't have to make a change in MetaPackages (cc @JunTaoLuo )

Reply:
Seems like this says exclude, so no changes to MetaPackage is required.

Reply:
@rynowak should this be `shipoob` or `noship`?

Reply:
`shipoob` - this package and a few others from Razor ship in VS **only**